### ISSUE TITLE
Typo on Workflows File

### ISSUE THREAD

**Issue by BlueWings98** [MelbourneDeveloper/RestClient.Net#137]
There is a Typo on the workflows file that may suggest that those tests are not being run.
`      - name: Test on .NET Core 3.1
        run: dotnet test src/RestClient.Net.UnitTests --no-build --verbosity normal --framework nercoreapp3.1
`
Instead of "**ner**coreapp3.1" it should say "**net**coreapp3.1"
This is of great importance as it ensures compatibility with that legacy code.

**Comment by MelbourneDeveloper** [comment]
@BlueWings98 thanks. The pipelines currently don't run. This library has been on standstill for a while. I'm hoping to do a new version soon, and when that happens, I'll run the tests on GitHub actions.

**Comment by BlueWings98** [comment]
@MelbourneDeveloper I think the best next move would be to try to make this project more open-source friendly, that way you wouldnt be alone developing this. Creating a better readme would be a great first step.

**Comment by MelbourneDeveloper** [comment]
@BlueWings98 what's the issue with the readme?

**Comment by BlueWings98** [comment]
@MelbourneDeveloper 
First, one of the images link is broken.
<img width="255" alt="image" src="https://github.com/user-attachments/assets/62eb456d-030a-405f-8220-b7cf45d2906d">

Doesn't really tell you how to install and run the project.
The contribution section is hidden at the very end and could be expanded.
It would be very helpful to have a rundown of the design of the project, to understand how everything is organized, specially because an analizing software that I used picked up many projects within the solution. Many of them just generating noise. 
**TLDR:** If we want people to help the project we need to make them unsderstand the project easiest as possible, in order to reduce the cognitive complexity.
